packages/vue-pdf-viewer/src/components/PDFViewer.vue

```diff
@@ -58,9 +58,11 @@ export default {
             type: String,
             required: true
         },
-        url: {
-            type: String,
-            required: true
+        source: {
+            required: true,
+            validator: (value) => {
+                return !!value && (typeof value === 'string' || value.constructor === Uint8Array)
```

Comment:
Does anyone see a problem in just checking for Uint8Array when a blob is used? Couldn't find helpful info about it

Reply:
Blob and Uint8Array are two different things in javascript. As long as you expect to always pass a Uint8Array as source, it should be fine. To read an Uint8Array from a Blob you would need a FileReader: https://developer.mozilla.org/de/docs/Web/API/Blob

Reply:
But instead of `value.constructor`, I would check it with `value instanceof Uint8Array`

Reply:
It's always a Uint8Array but the property is called blob. I updated the validator with `value instanceof Uint8Array`